core/ai_worker.go

```diff
@@ -375,6 +381,14 @@ func (rw *RemoteAIWorker) Process(logCtx context.Context, pipeline string, model
 			monitor.AIResultDownloaded(logCtx, pipeline, modelID, chanData.DownloadTime)
 		}
 
+		monitor.SendQueueEvent("ai_worker_job_sent", map[string]string{
```

Comment:
"job_sent" is this the worker sending a job to the runner or the gateway sending the job to the worker?